app/controllers/trajectories_controller.rb

```diff
@@ -0,0 +1,54 @@
+class TrajectoriesController < ApplicationController
+    before_action :set_trajectory, only: %i[show create update destroy]
+
+  
+    def show
+      @trajectory = Trajectory.includes(:schedules).find(params[:id])
```

Comment:
mmmm IMHO this should be in the Trajectory model

Reply:
It depends if we have it for specific cases, it could go in the model.